packages/react/src/components/Message/MessageToolbox.js

```diff
@@ -1,11 +1,17 @@
-import React, { useState } from 'react';
+import React, { useContext, useState } from 'react';
 import { css } from '@emotion/react';
 import Popup from 'reactjs-popup';
 import useComponentOverrides from '../../theme/useComponentOverrides';
 import { Box } from '../Box';
 import { appendClassNames } from '../../lib/appendClassNames';
 import { ActionButton } from '../ActionButton';
 import { EmojiPicker } from '../EmojiPicker';
+import { Modal } from '../Modal';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+import RCContext from '../../context/RCInstance';
+import { useToastBarDispatch } from '../../hooks/useToastBarDispatch';
+import { useToastStore } from '../../store';
```

Comment:
This was removed recently and we are no longer using the store for toasts. I think this is the older code.